frontend_tests/node_tests/compose.js

```diff
@@ -554,6 +555,121 @@ people.add(bob);
     assert(compose_finished_event_checked);
 }());
 
+(function test_initialize() {
+    // In this test we mostly do the setup stuff in addition to testing the
+    // normal workflow of the function. All the tests for the on functions are
+    // done in subsequent tests directly below this test.
+
+    var resize_watch_manual_resize_checked = false;
+    resize.watch_manual_resize = function (elem) {
+        assert.equal('#new_message_content', elem);
+        resize_watch_manual_resize_checked = true;
+    };
+    global.window = {
+        XMLHttpRequest: true,
+        bridge: true,
+    };
+    var xmlhttprequest_checked = false;
+    set_global('XMLHttpRequest', function () {
+        this.upload = true;
+        xmlhttprequest_checked = true;
+    });
+    $("#compose #attach_files").addClass("notdisplayed");
+    feature_flags.dropbox_integration = true;
+    var dropbox_integration_loaded = false;
+    global.LazyLoad.js = function (script, on_success) {
+        assert.equal(script, 'https://www.dropbox.com/static/api/1/dropins.js');
+        dropbox_integration_loaded = true;
+
+        blueslip.error = function (error_msg) {
+            assert.equal(error_msg, 'Dropbox script reports loading but window.Dropbox undefined');
+        };
+        on_success();
+
+        global.window.Dropbox = 'fake-dropbox-script';
```

Comment:
You may need to call `set_global` in a few more places.  I would consider punting on the Dropbox stuff for now, since I think Cory may be working on it soon, and we don't even know if the real code works here any more, since the feature flag is off.